phpstan.neon.dist

```diff
@@ -9,6 +9,7 @@ parameters:
     excludes_analyse:
         - var
         - vendor
+        - tests/TraceContext/W3CTestService
```

Comment:
Excluding the test service directory here will avoid some erroneous phpstan errors. Some files used by the test service are dynamically created so a phpstan test run will show errors for these files that don't yet exist.